about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Us</title>
+    <link href="about.css" rel="stylesheet">
+</head>
+<body>
+<section class="wrapper">
+    <header>
+        <div id="nav-menu">
+            <div id="btn-close">&times;</div>
+            <ul>
+                <li><a href="#about" class="nav-link" onclick="clickX()">About</a></li>
+                <li><a href="#program" class="nav-link" onclick="clickX()">Program</a></li>
+                <li><a href="#join" class="nav-link" onclick="clickX()">Join</a></li>
+                <li><a href="#sponsor" class="nav-link" onclick="clickX()">Sponsor</a></li>
+                <li><a href="#news" class="nav-link" onclick="clickX()">News</a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <img class="ham" src="./Image/hamburger.png">
+        </nav>
```

Comment:
- [ ]  I noticed that you don't have a link back to the `Home Page` when on the `About Page` of the mobile view so kindly fix this as well :100: 
- Your design on the `About Page`
![image](https://user-images.githubusercontent.com/102699958/235146873-00ac6616-0efd-4caa-8f32-f0cb529243c0.png)